src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -253,7 +253,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Overrides Function GetImports() As ImmutableArray(Of String)
```

Comment:
This is how the C# implementation works. However CLI parser does not handle alias references and so global imports will not include them.
Also I have added filtering out of xml references.